### ISSUE TITLE
Terminal GIF out of sync with documentation

### ISSUE THREAD

**Issue by chrisanderss0n** [patternplate/patternplate#205]
The GIF in the documentation here:

https://patternplate.github.io/doc/docs/advanced/deploy?guides-enabled=true&advanced-enabled=true#build-a-static-interface

Is out of sync with the written documentation.

How was the GIF created? Happy to fix myself.

**Comment by marionebl** [comment]
Hey @chrisanderss0n, that is a good catch - thank you!

The sequence is a CSS-animated SVG created via [svg-term-cli](https://github.com/marionebl/svg-term-cli). 

The relevant repo that power `patternplate.github.com/media` is here: 
https://github.com/patternplate/media

The setup there works by using [casts/build/screencast.config.js](https://github.com/patternplate/media/blob/master/casts/build/screencast.config.js#L2-L9) to generate [docs/casts/cast-build.svg](https://github.com/patternplate/media/blob/master/docs/casts/cast-build.svg) whenever you execute `yarn cast` in a copy of the media repo. 

So, long story short: to change the SVG result you'll have to  change the `cast` string found in [casts/build/screencast.config.js](https://github.com/patternplate/media/blob/master/casts/build/screencast.config.js#L2-L9).

Thanks for offering your help, I really appreciate it.
